app/lib/world/world_script_note.coffee

```diff
@@ -11,7 +11,7 @@ module.exports = class WorldScriptNote
     @invalid = false
     @channel = script.channel
     @event ?= {}
-    @event.replacedNoteChain = script.noteChain
+    @event.replacedNoteChain = script.noteChain if script.noteChain
```

Comment:
This fixes a bug that was on Master where it would fail to send `world:thang-died` (or something) events because of a this value being `undefined` instead of missing.

Reply:
This should be fine, I don't even see anywhere this is used